server/controllers/article.controller.js

```diff
@@ -204,5 +204,20 @@ class ArticleController {
     await sendMail(mailOption);
     return Response(res, 200, 'Article shared successfully');
   }
+
+  /**
+   * description Get all tags
+   * @param {object} req
+   * @param {object} res
+   * @returns {object} response
+   */
+  static async getAllTags(req, res) {
+    const tags = await Tag.findAndCountAll({
+      attributes: ['name']
+    });
+    if (!tags) return Response(res, 404, 'No tags currently');
```

Comment:
@Chiazokam I think it would be okay to return `200` and an empty array when tags have not been created and the user is not looking for a specific tag. What do you think?

Reply:
Alright....will make this change

Reply:
Done...thanks